src/pages/components/table-of-contents.mdx

```diff
@@ -32,7 +32,15 @@ The Table of Contents (ToC) component renders as left navigation that includes a
 
 <Title>Use case</Title>
 
-![Image of Table of contents component](../../images/component/table-of-contents/table-of-content-default-use-case.jpg)
+![Image of Table of contents component usecase](../../images/component/table-of-contents/table-of-content-default-use-case.jpg)
+
+**When to use:**
+* When users need to know what is included in a page, mainly when it contains a log of information. It serves as an outline or summary of the page content.
```

Comment:
I wonder if the below bullet point covers this section?

Reply:
Are you suggesting to take " When users need to know what is included in a page, mainly when it contains a log of information. It serves as an outline or summary of the page content." off or to keep it but delete "When users need to rely on scrolling to consume the information on the page."?